Author: Bernhard M. Wiedemann <bwiedemann suse.de>
Date:   2020-03-26

https://bugzilla.opensuse.org/show_bug.cgi?id=1159231

avoid crashes on machines that are older than the random build machine
and make package builds reproducible.

See https://reproducible-builds.org/ for why this is good.

Index: tesseract-4.1.1/src/arch/Makefile.am
===================================================================
--- tesseract-4.1.1.orig/src/arch/Makefile.am
+++ tesseract-4.1.1/src/arch/Makefile.am
@@ -47,9 +47,6 @@ libtesseract_sse_la_CXXFLAGS = -msse4.1
 endif
 
 libtesseract_native_la_CXXFLAGS = -O3 -ffast-math
-if MARCH_NATIVE_OPT
-libtesseract_native_la_CXXFLAGS += -march=native -mtune=native
-endif
 libtesseract_native_la_SOURCES = dotproduct.cpp
 
 libtesseract_arch_la_SOURCES = intsimdmatrix.cpp simddetect.cpp
